include/rosdiscover-clang/ApiCall/Calls/Util.h

```diff
@@ -43,8 +43,8 @@ namespace rosdiscover {
     return false;
   }
 
-  static std::string prettyPrint(clang::Stmt* const statement, clang::ASTContext const &context) {
-    auto range = clang::CharSourceRange::getTokenRange(statement->getSourceRange());
+  static std::string prettyPrint(clang::SourceRange sourceRange, clang::ASTContext const &context) {
```

Comment:
This probably shouldn't be a static function. Also, there shouldn't be an indent before functions (or anything) inside a namespace. (This helps to improve readability; in this case, without looking at the surrounding context, it looks like a static method, but expanding the PR context shows it's actually a C-style static function.)

Reply:
Fixed